README.md

```diff
@@ -1,75 +1,79 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/Misheck12-blueviolet)
 
-# Project Name
+# Hello React Front End app
 
-> Description the project.
+> Hello React Front End app is simple application that contains setup of react application. The greeting page loads a random greeting from API on every page load.
 
+## Link to the back end
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+[Hello Rails back end](https://github.com/misheck12/hello_world_rails-react)
```

Comment:
- [ ] In this project you are supposed to connect Ruby on Rails back-end and React front-end using `jsbundling-rails`. Kindly check the project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md) and work on it.

Reply:
hello @uwadonat  thank you for your review. on this project, its supposed to be front-end only.  the backend and front end are connected in the second application which is submitted separately

Reply:
Hello @misheck12, is [this](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md) the project requirements you are Following?

Reply:
hello @uwadonat yes this is the requirement but there is a separate repository for the front-end and backend. my backend has been approved with a reference to the front end. maybe I should have added the back-end link before sending it for review. i have added the backend link in the pull request.

Reply:
Hi @misheck12. kindly check again the requirements. you are supposed to submit one project combining both rails and react part connected by  `jsbundling-rails gem`.